### PR TITLE
CEDS-2331 - Migrate 1st part of 'Add Items' section to Gov Design System

### PR DESCRIPTION
**Pull request by SteveSugden**
- procedure_codes
- taric_codes

### CODE REVIEW

File: app/views/declaration/procedure_codes.scala.html [hunk 1]
```diff
@@ -15,60 +15,94 @@
...
-                sectionHeader = messages("supplementary.items"),
-                title = messages("supplementary.procedureCodes.title")
-            )
+        @govukFieldset(Fieldset(
+            legend = Some(Legend(
+                content = Text(messages("declaration.procedureCodes.title")),
+                classes = gdsPageLegend,
+                isPageHeading = true
+            )),
+            html = HtmlFormat.fill(immutable.Seq(
```
Comment (MaciejRewera) [inline app/views/declaration/procedure_codes.scala.html:87]: Nit: `immutable` is redundant here.

File: app/views/declaration/taric_codes.scala.html [hunk 2]
```diff
@@ -43,43 +60,67 @@
...
-) {
+@pageHint = {
+    <span class="govuk-hint">
+        @Html(messages("declaration.taricAdditionalCodes.header.hint", tradeTariffLink))
+    </span>
+}
 
-    @helper.form(routes.TaricCodeController.submitForm(mode, itemId), 'autocomplete -> "off") {
-        @helper.CSRF.formField
+@taricCodesTable = {
```
Comment (MaciejRewera) [inline app/views/declaration/taric_codes.scala.html:69]: Have you considered making this a component? Similar pattern is repeated in several places.

File: conf/messages.en [hunk 3]
```diff
@@ -503,22 +503,23 @@ declaration.goodsLocation.code.empty = Goods location code cannot be empty
...
+declaration.procedureCodes.additionalProcedureCode.header = Do you need to enter additional procedure codes?
+declaration.procedureCodes.additionalProcedureCode.header.hint = A 3 digit code. Up to 99 may be declared for each goods item.
+declaration.procedureCodes.additionalProcedureCode.item.header = 1/11 Additional procedure codes
+declaration.procedureCodes.additionalProcedureCode.error.invalid = Enter an additional procedure code in the correct format
+declaration.procedureCodes.additionalProcedureCode.mandatory.error = Enter an additional procedure code in the correct format
+declaration.procedureCodes.additionalProcedureCode.maximumAmount.error = Maximum amount of additional procedure codes added
+declaration.procedureCodes.additionalProcedureCode.empty = Enter an additional procedure code in the correct format
+declaration.procedureCodes.additionalProcedureCode.duplication = Additional procedure code already added
+declaration.procedureCodes.additionalProcedureCode.remove.hint = Remove additional code {0}
+declaration.procedureCodes.additionalProcedureCode.add.hint = Add additional code
```
Comment (MaciejRewera) [inline conf/messages.en:522]: Big 👍

File: app/views/declaration/procedure_codes.scala.html (hunk 1 above)
Comment (SteveSugden) [reply app/views/declaration/procedure_codes.scala.html:87]: Doesn't compile without it.  There's 2x types of Seq on this page

File: app/views/declaration/taric_codes.scala.html (hunk 2 above)
Comment (SteveSugden) [reply app/views/declaration/taric_codes.scala.html:69]: Possibly.   But by the time you've discovered all of the similar places where that component could be re-used it's often simpler not to bother.

File: conf/messages.en (hunk 3 above)
Comment (SteveSugden) [reply conf/messages.en:522]: Thanks

**Comment by SteveSugden** [comment]
Yes - I try not to break any tests if I can help it.  Gives me re-assurance that the conversion is successful.

**Pull request merged**